Yoast/wpseo-news
# Fixed version update

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where the grunt version-update would not update the version number in `wpseo-news.php`.
* Introduces the use of the yoast-grunt tasks grunt plugin.

## Relevant technical choices:

* Using the github url in package.json

## Test instructions

The Grunt commands set-version and update-version is the main way to change the version numbers over the various files in the plugin.

This PR can be tested by following these steps:

1. running `yarn`
2. Running `grunt set-version   --new-version=1`
3. Running `grunt update-version`
verify if the version is set to the argument given on step 2 in the following files:
    * README.md
    * wpseo-news.php
    * classes/class-wpseo-news.php

Fixes #399
